packages/react-microlink/src/components/Card/CardMedia/video.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import styled, {css} from 'styled-components'
+
+import { media, isLarge } from '../../../utils'
+
+const largeStyle = css`
+flex: 1;
+
+&::before {
+  padding-bottom: 0;
+}
+`
+
+const VideoWrapper = styled.div`
+  display: block;
+  flex: 0 0 125px;
+  overflow: hidden;
+  height: auto;
+  position: relative;
+
+  &::before {
+    content: '';
+    padding-bottom: 100%;
+    display: block;
+  }
+
+  ${({cardSize}) => isLarge(cardSize) ? largeStyle : media.mobile`
+    flex: 0 0 92px;
+  `}
+`
+
+const Video = styled.video`
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 100;
+
+  ${({autoPlay}) => autoPlay && `
+    &::-webkit-media-controls-start-playback-button {
+      display: none!important;
+      -webkit-appearance: none;
```

Comment:
what's that 🤔

Reply:
It hides the typical ▶️ button from the `<video />` element, ie. this:

![image](https://user-images.githubusercontent.com/5795227/35943326-ee5e7eae-0c58-11e8-810e-ec9e7671ad9a.png)

It was displaying over the videos even if `autoplay` was enabled, so I've hidden it via CSS when `autoPlay === true`

Reply:
👌